service/src/main/java/uk/gov/hmcts/reform/fpl/service/LocalAuthorityUserService.java

```diff
@@ -20,11 +20,17 @@
     private static final Set<CaseRole> CASE_ROLES = Set.of(LASOLICITOR, CREATOR);
     private final CaseRoleService caseRoleService;
     private final RequestData requestData;
+    private final FeatureToggleService featureToggleService;
 
     public void grantUserAccessWithCaseRole(String caseId, String caseLocalAuthority) {
         String currentUser = requestData.userId();
-
-        caseRoleService.grantAccessToLocalAuthority(caseId, caseLocalAuthority, CASE_ROLES, Set.of(currentUser));
-        caseRoleService.grantAccessToUser(caseId, currentUser, CASE_ROLES);
+        if (null != featureToggleService && featureToggleService.isCaseUserAssignmentEnabled()) {
```

Comment:
featureToggleService wont be null, application wont bootstrap if it cannot be created

Reply:
I have executed test under LocalAuthorityUserServiceTest :: shouldGrantCaseRolesToAllLocalAuthorityUsers 
that failed on null featuretoggle , hence I have added this check ... thoughts?

Reply:
You introduced new dependency, if you see other dependencies like *CaseRoleService* are mocked in the test. You need to add new mock for your new (FeatureToggleService) dependency as well.

Reply:
got it.. Thanks Adam